kubernetes/apps/base/api/deployment.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       serviceAccountName: api
       containers:
         - name: api
-          image: northamerica-northeast1-docker.pkg.dev/pdcp-cloud-005-safeinputs/safeinputs/api:main-b439424-1719438069 # {"$imagepolicy": "flux-system:api"}
```

Comment:
@vedantthapa I assume we'll either want to get flux to include the digest sha when updating images, or we'll have to tell renovate to not touch `deployment.yaml`s.